openwrt/README.md

```diff
@@ -12,6 +12,24 @@ If you are building images with templates you'll also need:
 * [gomplate](../README.md#requirements)
 
 ## Build
+
+### Docker
+
+You can build these images inside a docker container. This makes it easy to has a consistent build environment
+for all members of the tech team.
+
+To start building:
+
+```
+docker pull sarcasticadmin/openwrt-build:528bc79
+# Make sure to mount the git root inside this container
+docker run -v $(git rev-parse --show-toplevel):/home/openwrt/scale-network --rm -it sarcasticadmin/openwrt-build:528bc79 /bin/bash
```

Comment:
works as advertised, very cool

Reply:
thanks! the idea was to keep the build cache between containers since its large 10GB+